rewrite-java/src/main/java/org/openrewrite/java/tree/Space.java

```diff
@@ -444,7 +445,17 @@ public enum Location {
         WHILE_PREFIX,
         WILDCARD_BOUND,
         WILDCARD_PREFIX,
-        YIELD_PREFIX,
+        YIELD_PREFIX;
+
+        @Incubating(since = "8.11.3")
+        public boolean isPrefix() {
+            return this.toString().endsWith("_PREFIX");
```

Comment:
Do we get a lot of false positives without this?

Reply:
The logic does not work correctly without a check to see if the `Location` is a prefix.

Without it, we end up findings AST elements on other lines